### ISSUE TITLE
[Pylint] pytest.mark.parametrize is not callable (not-callable)

### ISSUE THREAD

**Issue by kaushikcfd** [inducer/ci-support#1]
Pylint CIs are failing with [pytest-6.0.0](https://pypi.org/project/pytest/6.0.0/) (c.f. https://github.com/inducer/pytools/pull/29/checks?check_run_id=925808944).

Quite strange because:
```
In [1]: import pytest                                      

In [2]: pytest.__version__                                 
Out[2]: '6.0.0'

In [3]: callable(pytest.mark.parametrize)                  
Out[3]: True
``` 

Pinning `"pytest<6"` in `prepare-and-run-pylint.sh` solves the issue.

**Comment by kaushikcfd** [comment]
Related: https://github.com/pytest-dev/pytest/pull/7565

**Comment by kaushikcfd** [comment]
Fixed with the release of [pytest-6.0.1](https://pypi.org/project/pytest/6.0.1/).
